draft-ietf-webtrans-http3.md

```diff
@@ -294,11 +294,23 @@ SHALL include a single choice from the client's list in that field. Servers MAY
 reject the request if the client did not include a suitable protocol.
 
 Both `WT-Available-Protocols` and `WT-Protocol` are Structured Fields
-{{!RFC8941}}. `WT-Available-Protocols` is a List of Tokens, and `WT-Protocol` is
-a Token. The token in the `WT-Protocol` response header field MUST be one of
-the tokens listed in `WT-Available-Protocols` of the request.  The semantics of
-individual token values is determined by the WebTransport resource in question
-and are not registered in IANA's "ALPN Protocol IDs" registry.
+{{!FIELDS=RFC9651}}. `WT-Available-Protocols` is a List.  `WT-Protocol` is
+defined as an Item. In both cases, the only valid value types are String or
+Token. Any value type other than String or Token MUST be treated as an error
+that causes the entire field to be ignored.  String or Token values can be used
+interchangeably, but endpoints SHOULD use String; see {{Section 3.3.4 of
+!FIELDS}}.  Decoded values are used to identify protocols. No semantics are
+defined for parameters on either field; parameters MUST be ignored.
+
+The value in the `WT-Protocol` response header field MUST be one of the values
+listed in `WT-Available-Protocols` of the request.  Otherwise, the `WT-Protocol`
```

Comment:
This seems to be a restatement (albeit a better one) of the requirement on L293

>  If the server receives such a header, it MAY include a
`WT-Protocol` field in a successful (2xx) response. If it does, the server
SHALL include a single choice from the client's list in that field.

Should we just remove the earlier one?

Reply:
Note that the first MAY is not redundant (you're allowed to not send any response).